app/extlinux_control.py

```diff
@@ -348,7 +348,7 @@ def _replace(ma: re.Match, repl: str):
             return res
 
         _repl_func = partial(_replace, repl=f"root={self.standby_slot_partuuid}")
-        dst.write_text(re.compile(r"\n\sAPPEND.*").sub(_repl_func, src.read_text()))
```

Comment:
Emmm you just forgot to add * after \s, no need to change back to use `.*APPEND.*`.